webpack/published-ts-components.js

```diff
@@ -1,6 +1,7 @@
 const publishedTSComponents = {
   // Don't remove next line
   // plop_marker:published-components
+  Badge: "components/Badge/Badge",
```

Comment:
name-wize, I think that we wanted to call it `NotifiationBadge`.
I agree that it is generic enough to be called `Badge`, but a `Badge` can be also textual.
we might even go with `BulletBadge`,

Reply:
Actually the `Badge` is the component that responsible for "layouting" types of badges on a child component, so basically we can attach whatever we want to it, even if it is textual
You want to re-discuss the naming in a wider forum? since Yael has decided to go with Badge and I don't want to change something without her knowing